Formula/alp.rb

```diff
@@ -0,0 +1,44 @@
+class Alp < Formula
+  desc "Access Log Profiler"
+  homepage "https://github.com/tkuchiki/alp"
+  url "https://github.com/tkuchiki/alp/archive/v1.0.1.tar.gz"
+  sha256 "7206f60efcca9a4f149d6abdcaf806f290fcf16237b1f7528e40628c37816449"
+  head "https://github.com/tkuchiki/alp.git"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    srcpath = buildpath/"src/github.com/tkuchiki/alp"
+    srcpath.install buildpath.children
+
+    cd srcpath do
+      system "dep", "ensure", "-vendor-only"
+      system "go", "build", "-o", bin/"alp", "cli/alp/main.go"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    (testpath/"json_access.log").write <<~EOS
+      {"time":"2015-09-06T05:58:05+09:00","method":"POST","uri":"/foo/bar?token=xxx&uuid=1234","status":200,"body_bytes":12,"response_time":0.057}
+      {"time":"2015-09-06T05:58:41+09:00","method":"POST","uri":"/foo/bar?token=yyy","status":200,"body_bytes":34,"response_time":0.100}
+      {"time":"2015-09-06T06:00:42+09:00","method":"GET","uri":"/foo/bar?token=zzz","status":200,"body_bytes":56,"response_time":0.123}
+      {"time":"2015-09-06T06:00:43+09:00","method":"GET","uri":"/foo/bar","status":400,"body_bytes":15,"response_time":"-"}
+      {"time":"2015-09-06T05:58:44+09:00","method":"POST","uri":"/foo/bar?token=yyy","status":200,"body_bytes":34,"response_time":0.234}
+      {"time":"2015-09-06T05:58:44+09:00","method":"POST","uri":"/hoge/piyo?id=yyy","status":200,"body_bytes":34,"response_time":0.234}
+      {"time":"2015-09-06T05:58:05+09:00","method":"POST","uri":"/foo/bar?token=xxx&uuid=1234","status":200,"body_bytes":12,"response_time":0.057}
+      {"time":"2015-09-06T05:58:41+09:00","method":"POST","uri":"/foo/bar?token=yyy","status":200,"body_bytes":34,"response_time":0.100}
+      {"time":"2015-09-06T06:00:42+09:00","method":"GET","uri":"/foo/bar?token=zzz","status":200,"body_bytes":56,"response_time":0.123}
+      {"time":"2015-09-06T06:00:43+09:00","method":"GET","uri":"/foo/bar","status":400,"body_bytes":15,"response_time":"-"}
+      {"time":"2015-09-06T06:00:43+09:00","method":"GET","uri":"/diary/entry/1234","status":200,"body_bytes":15,"response_time":0.135}
+      {"time":"2015-09-06T06:00:43+09:00","method":"GET","uri":"/diary/entry/5678","status":200,"body_bytes":30,"response_time":0.432}
+      {"time":"2015-09-06T06:00:43+09:00","method":"GET","uri":"/foo/bar/5xx","status":504,"body_bytes":15,"response_time":60.000}
+      {"time":"2015-09-06T06:00:43+09:00","method":"GET","uri":"/req","status":200,"body_bytes":15,"response_time":"-", "request_time":0.321}
+    EOS
+    system "#{bin}/alp", "json", "--file=#{testpath}/json_access.log", "--dump=#{testpath}/dump.yml"
```

Comment:
Can we also assert that this didn't return with a non-zero status?

Reply:
@SMillerDev Yes, if the file is not found or incorrect, this returns a non-zero status code.

Reply:
@SMillerDev 
![Alacritty 2019-09-11 at 19 45 00@2x](https://user-images.githubusercontent.com/38746192/64690994-b06f1e80-d4cc-11e9-86d2-3dff06289f36.png)

Reply:
That's fine, but can you make the block test for it.

Reply:
Sorry, what does the block test mean? If this line fails, the test also fails. Is this insufficient?